source/tests/system/nifgen_driver_api_tests.cpp

```diff
@@ -588,6 +585,9 @@ TEST_F(NiFgenDriverApiTest, ExportTriggerMode_ResetAndImportConfiguration_Trigge
 
 TEST_F(NiFgenDriverApiTest, AllocateWaveform_WriteWaveformComplexF64_WaveformWrittenSuccessfully)
 {
+#ifndef WIN32
```

Comment:
Note: Found out this test won't work on Linux but the rest should. Failed on Linux when I didn't have this skip and looked at the [NI-FGEN documentation](https://www.ni.com/pdf/manuals/ni-fgen-linux-2023-q1.html) which mentioned OSP functionality wasn't supported on Linux so it made sense.